viadot/sources/databricks.py

```diff
@@ -199,18 +201,21 @@ def run(
 
         return result
 
-    def _check_if_table_exists(self, schema: str, table: str) -> bool:
+    def _check_if_table_exists(self, table: str, schema: str = None) -> bool:
+        if schema is None:
+            schema = Databricks.DEFAULT_SCHEMA
```

Comment:
This depends on `Databricks.DEFAULT_SCHEMA` which is not defined

Reply:
Kindly refer to line 43

Reply:
Ah nice, I missed it. While we're at it though you can remove the other two variables from there (`env` is already provided in `__init__` and `session` should be initialized in `__init__` as well):
```python
        self.env = env
        self.credentials = credentials or local_config.get("DATABRICKS", {}).get(env)
        self.session = None

        self.connect()
```